x/participationrewards/keeper/rewards_validatorSelection.go

```diff
@@ -17,7 +17,7 @@ import (
 // individually in a callback.
 func (k Keeper) AllocateValidatorSelectionRewards(ctx sdk.Context) {
 	k.icsKeeper.IterateZones(ctx, func(_ int64, zone *icstypes.Zone) (stop bool) {
```

Comment:
The PR score logic will only execute for baseChains. Might have to change how this is done, as currently it it computed on the fly for each zone - but in reality it need to be computed once for each baseChain and the scores reused for the subzone. This can wait until PR#2 though.